### PR TITLE
chore: [IOBP-985] Add onboarding payment method success mixpanel analytics event

### PR DESCRIPTION
**Pull request by Hantex9**
## Short description
This PR adds a mixpanel analytics event dispatched when the user onboard successfully a payment method

## List of changes proposed in this pull request
- Added the `analytics` folder inside the `ts/features/payments/onboarding` sub-feature with its dispatcher method `trackSuccessOnboardingPaymentMethod`;
- Added the logic inside the `PaymentsOnboardingFeedbackScreen.tsx` to track the onboarded payment method selected;

## How to test
- Start the onboarding payment method flow from the Wallet screen;
- Check that when you complete the onboarding successfully the mixpanel event is emitted

### CODE REVIEW

File: ts/features/payments/onboarding/screens/PaymentsOnboardingFeedbackScreen.tsx [hunk 1]
```diff
@@ -61,6 +73,17 @@ const PaymentsOnboardingFeedbackScreen = () => {
     Object.values(WalletOnboardingOutcomeEnum).indexOf(outcome)
   ] as keyof typeof WalletOnboardingOutcomeEnum;
 
+  useOnFirstRender(() => {
+    const onboardedPaymentMethodName = availablePaymentMethods?.find(
+      paymentMethod => paymentMethod.id === selectedPaymentMethodId
+    )?.name;
+    if (outcome === WalletOnboardingOutcomeEnum.SUCCESS) {
+      analytics.trackSuccessOnboardingPaymentMethod({
+        payment_method_selected: onboardedPaymentMethodName
```
Comment (LeleDallas) [inline ts/features/payments/onboarding/screens/PaymentsOnboardingFeedbackScreen.tsx:82]: You can pass this as shorthand

**Review by LeleDallas — COMMENTED** [review]
Positively tested💫
Here's an adjustment

File: ts/features/payments/onboarding/screens/PaymentsOnboardingFeedbackScreen.tsx (hunk 1 above)
Comment (Hantex9) [reply ts/features/payments/onboarding/screens/PaymentsOnboardingFeedbackScreen.tsx:82]: Are you referring to the onboardedPaymentMethodName? This constant is valuable in another iteration where tracking failure outcomes is necessary, ensuring that each outcome includes the payment_method_selected. I created it to have a readily available constant containing this information, making it reusable across other analytics methods that will be implemented in the next iteration as highlighted in this [jira comment](https://pagopa.atlassian.net/browse/IOBP-985?focusedCommentId=187609)

Comment (LeleDallas) [reply ts/features/payments/onboarding/screens/PaymentsOnboardingFeedbackScreen.tsx:82]: I mean you can rename it as `payment_method_selected` and then below pass it as shorthand. What do you think?

Comment (Hantex9) [reply ts/features/payments/onboarding/screens/PaymentsOnboardingFeedbackScreen.tsx]: It makes sense! Addressed into 42ce8deac8fe84b6d478045469f04aaa32a5f14a

**Review by LeleDallas — APPROVED** [review]

**Pull request merged**